generators/server/templates/src/main/java/package/config/SecurityConfiguration.java.ejs

```diff
@@ -217,10 +217,8 @@ public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
         <%_ } _%>
     <%_ } _%>
             .exceptionHandling()
-    <%_ if (authenticationType !== 'oauth2') { _%>
-            .authenticationEntryPoint(problemSupport)
-    <%_ } _%>
-            .accessDeniedHandler(problemSupport)
+                .authenticationEntryPoint(problemSupport)
```

Comment:
I think the indent is not correct

Reply:
`exceptionHandling()` method returns a reference of `ExceptionHandlingConfigurer` and `authenticationEntryPoint()` and `accessDeniedHandler()` are defined in that class. IMO, this is correct indentation.